src/pages/workspaces/workspace/Tools.js

```diff
@@ -269,15 +290,24 @@ export const Tools = _.flow(
     return _.find({ namespace, name }, configs)
   }
 
+  addRedactedAttribute(config, methods) {
```

Comment:
nit: we could read `methods` out of state rather than taking it as a param. Also, this name is a bit outdated, since we're no longer 'adding' anything. Maybe 'computeRedacted'?